pom.xml

```diff
@@ -35,6 +39,13 @@
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>git</artifactId>
             <version>2.3</version>
+            <exclusions>
+                <!-- To avoid ClassNotFoundException during InjectedTest -->
+                <exclusion>
+                    <groupId>org.apache.httpcomponents</groupId>
+                    <artifactId>httpclient</artifactId>
+                </exclusion>
+            </exclusions>
```

Comment:
It seems @jglick had to do the same [here](https://github.com/jenkinsci/workflow-plugin/blob/master/aggregator/pom.xml#L102).